frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/GraphTableDialog.kt

```diff
@@ -0,0 +1,107 @@
+package com.virtuslab.gitmachete.frontend.actions.dialogs
```

Comment:
Given that there's quite a few non-trivial changes here... pls bump the version to v0.7.1 (and I'll move issue/PR now)